tests/end-to-end/regression/5592-process-isolation-events.phpt

```diff
@@ -0,0 +1,69 @@
+--TEST--
+https://github.com/sebastianbergmann/phpunit/pull/5592
+--FILE--
+<?php declare(strict_types=1);
+$_SERVER['argv'][] = '--do-not-cache-result';
+$_SERVER['argv'][] = '--no-configuration';
+$_SERVER['argv'][] = '--process-isolation';
+$_SERVER['argv'][] = '--debug';
+$_SERVER['argv'][] = __DIR__ . '/5592/Issue5592Test.php';
```

Comment:
same test as in `tests/end-to-end/regression/5592-process-isolation.phpt`, but using `--debug`.

otherwise no changes